app/assets/javascripts/darkswarm/services/enterprise_registration_service.js.coffee

```diff
@@ -55,6 +55,10 @@ angular.module('Darkswarm').factory "EnterpriseRegistrationService", ($http, Reg
       ).catch((response) ->
         Loading.clear()
         alert(t('failed_to_update_enterprise_unknown'))
```

Comment:
Suprisly, we don't have any form errors displaying on that case. Only a popup alert.